server/api/min_resolved_ts_test.go

```diff
@@ -116,6 +119,28 @@ func (suite *minResolvedTSTestSuite) TestMinResolvedTS() {
 	})
 }
 
+func (suite *minResolvedTSTestSuite) TestMinResolvedTSByStores() {
+	// default run job
+	interval := typeutil.Duration{Duration: suite.defaultInterval}
+	suite.setMinResolvedTSPersistenceInterval(interval)
+	suite.Eventually(func() bool {
+		return interval == suite.svr.GetRaftCluster().GetPDServerConfig().MinResolvedTSPersistenceInterval
+	}, time.Second*10, time.Millisecond*20)
+	// set min resolved ts
+	rc := suite.svr.GetRaftCluster()
+	ts := uint64(233)
+	rc.SetMinResolvedTS(1, ts)
+	storeIDs := []string{"1"}
```

Comment:
What about mocking more stores to test all cases?

Reply:
done.